app/Http/Controllers/ChartController.php

```diff
@@ -170,6 +176,24 @@ public function getChart($symbol) {
         $totalRecentData = count($myRecentData);
         
         if ($totalRecentData > 0) {
+            $ema_short_period = 12;
+            $ema_long_period = 26;
+            $signal_period = 9;
+            
+            if( Auth::check()) {
+                $symbols = DB::table('configuremacdbots')->select("*")
+                        ->where('userid', '=', Auth::user()->id)
+                        ->where('symbol', '=', $symbol)
+                        ->whereNull('deleted_at')
+                        ->get();
+                if(count($symbols) > 0) {
+                    $ema_short_period = $symbols[0]->ema_short_period;
+                    $ema_long_period = $symbols[0]->ema_long_period;
+                    $signal_period = $symbols[0]->signal_period;
+                }
+            }
+            
+            
             array_walk($myRecentData, function(&$item, $key) {
                 $timeZoneOffSet = 3.30 * 3600;
```

Comment:
What is this doing?

Reply:
We are receiving dates in UTC format so, adding NL timezone offset to matching Dutch timezone.